djangoplicity/science/models.py

```diff
@@ -42,6 +41,11 @@
 from djangoplicity.media.models import Image
 from djangoplicity.translation.models import TranslationModel
 
+import django
+if django.VERSION >= (2, 2):
```

Comment:
Same comment as before, `django.core.urlresolvers` was removed in django 2.0.

Reply:
Well. It's done